web_tree_date_search/__openerp__.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```

Comment:
It would be good to shorten this copyright headers like in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py